apps/omg_jsonrpc/lib/server.ex

```diff
@@ -14,13 +14,16 @@
 
 defmodule OMG.JSONRPC.Server.Handler do
   @moduledoc """
-  Exposes OMG.API via jsonrpc 2.0 over HTTP. It leverages the generic OMG.JSONRPC.Exposer convenience module
+  Exposes an API via jsonrpc 2.0 over HTTP. It leverages the generic OMG.JSONRPC.Exposer convenience module
 
   Only handles the integration with the JSONRPC2 package
   """
   use JSONRPC2.Server.Handler
 
+  # Running :omg_jsonrpc requires setting this environmental variable
+  @api_module Application.fetch_env!(:omg_jsonrpc, :api_module)
```

Comment:
I believe this one is broken. I.e. Application.fetch_env is supposed to work in runtime, not in compile time. This effectively reads value from files that exist on dev machine during compilation (mix.exs or config.exs) and will ignore whatever way we will use to provide configuration to packaged release.

Reply:
You can test it by running `iex -S mix run --no-start` and changing the value of env variable via :application.set_env

Reply:
actually, we gave it a moment of thought and realised, we want it to be a compile-time constant in fact. Does this make it easier to live with?

Reply:
Yeah, this works fine. Commenting that such usage is intended would be good to make reading simpler.